test_discovery.py

```diff
@@ -96,12 +113,33 @@ def test_one(test_dict, intent_whitelist, domain_whitelist):
         test_dict, resp, VERBOSE_LOGGING
     )
 
-    intent_results["expected_entities"] = test_dict
-    intent_results["observed_entities"] = entity_results['observed_entity_dict']
+    intent_results["expected_entities"] = {
+        k: v
+        for k, v in test_dict.items()
+        if k not in ['test', 'transcript', 'schema', 'intent']
+    }
+    intent_results["observed_entities"] = \
+        entity_results['observed_entity_dict']
+
+    if 'schema' in test_dict:
+        intent_results = extract_schema_results_as_entities(
+            test_dict, intent_results, entity_results
+        )
 
     return y_true, y_pred, time_dif_ms, intent_results, entity_results
 
 
+def did_test_fail(entity_results, intent_results):
+    return (
+        1 if (
+          'expected_intent' in intent_results and \
+          intent_results['expected_intent'] != intent_results['observed_intent'] \
+        ) or ( \
```

Comment:
don't think the `\` are necessary as it is in parantheses

Reply:
@ageojo this overrides `yapf` when `\` is present

Reply:
Yep, as Matthew said this is just to prevent a strange yapf I saw